packages/gasket-request/README.md

```diff
@@ -0,0 +1,130 @@
+# @gasket/request
+
+The purpose of this package is to provide a consistent request object for Gasket
+plugins and apps to use, regardless of the request handling framework.
+
+## Installation
+
+#### Existing apps
+
+```shell
+npm install @gasket/request
+```
+
+## Functions
+
+### GasketRequest
+
+The `GasketRequest` class is a representation of a request object that can be
+used by Gasket plugins and apps. It is a consistent shape that can be used
+across different request handling frameworks.
+
+A `GasketRequest` object has the following properties:
+- `headers` - (object) Request headers
+- [`cookies`] - (object) Request cookies
+- [`query`] - (object) Query parameters
+- [`path`] - (string) Request path
+
+A `GasketRequest` can be created from a Node `IncomingMessage` object, an Express
+`Request` object, or a Next.js `NextRequest` object, amongst others.
+
+```js
+import { GasketRequest } from '@gasket/request';
+
+export default async function handler(req, res) {
+  const gasketRequest = new GasketRequest.make(req);
```

Comment:
Are these typos? I can see `new GasketRequest(...)` or `GasketRequest.make(...)` but not both in conjunction?

Reply:
That's a typo